src/codebook/summaryTable/renderRow/makeDetails.js

```diff
@@ -25,6 +25,17 @@ export default function makeDetails(d) {
       .attr("class", "stat");
     stats.append("div").text(d => d.key).attr("class", "label");
     stats.append("div").text(d => d.value).attr("class", "value");
+	// Display Unique Values for Categorical Variables
```

Comment:
This works, but there's an easier way.  Just add the number of distinct levels to the summary data object for categorical variables in [this method](d3.set](https://github.com/RhoInc/web-codebook/blob/181bf34cd1715ec8078a3f5dfaaba1c7332fa2c2/src/codebook/data/makeSummary.js) using [d3.set](https://github.com/RhoInc/web-codebook/blob/181bf34cd1715ec8078a3f5dfaaba1c7332fa2c2/src/codebook/data/makeSummary.js#L17)

Once that variable is added to the data object, the new 'unique' statistic should renderer automatically without any updates to this file.